lib/activerecord-import/import.rb

```diff
@@ -37,18 +36,21 @@ def valid_model?(model)
       model.send(:validation_context=, validation_context)
       model.errors.clear
 
-      validation_proc = lambda do
-        @validators.each do |v|
-          if validation_context == v.options.fetch(:on, validation_context)
-            v.validate(model) if validate?(v, model)
-          end
-        end
+      validate_callbacks = model._validate_callbacks
+      validate_callbacks.each do |callback|
+        validate_callbacks.delete(callback) if callback.raw_filter.is_a? ActiveRecord::Validations::UniquenessValidator
       end
 
-      if model.respond_to?(:run_validation_callbacks)
-        model.send(:_run_validation_callbacks, &validation_proc)
-      else
-        model.send(:run_callbacks, :validation, &validation_proc)
+      model.run_callbacks(:validation) do
+        if defined?(ActiveSupport::Callbacks::Filters::Environment) # ActiveRecord >= 4.1
```

Comment:
Is this safe? Or is there any possibility that `ActiveSupport::Callbacks::Filters::Environment` is not defined because it has not yet been auto-loaded?